packages/ui/__stories__/SelectField.stories.tsx

```diff
@@ -300,6 +301,56 @@ export const NonSearchableOpen = () => {
   )
 }
 
+export const MultipleSelections = () => {
+  const [currentValue, setValue] = useState([options[1], options[2]])
+  return (
+    <SelectField
+      name={name}
+      value={currentValue}
+      isMulti={true}
+      label={label}
+      options={options}
+      onBlur={() => logger.log('blur')}
+      onChange={setValue}
+    />
+  )
+}
+
+export const MultipleSelectionsMultipleRows = () => {
+  const [currentValue, setValue] = useState([...options])
+  return (
+    <SelectField
```

Comment:
Should we wrap it with a `div` with some max-wdith, say, 400px, to make it really multiple rows? On the current screenshot it is only one row